FlaskSite/controllers/info_controller.py

```diff
@@ -38,3 +64,30 @@ def update_info():
 
 def delete_info():
     pass
+
+def cleanup_list(input_list):
+    input_list = get_unique_list_of_dictionaries(input_list)
+    input_list = remove_in_complete_input(input_list)
+    return input_list
+
+def remove_in_complete_input(input_list):
+    """
+        take a list of dictionaries,
+        this method responsibility is to check for each dictionary in the list
+        it contains a value for a predefined key from must_exist_values
+        key = headline
+        if a dictionary contains key = headline, so it must be not empty, spaces or None
+    """
+    must_exist_values = ['headline', 'text', 'path', 'header', 'link']
+    new_list = []
+    print(input_list)
+    for current_dic in input_list:
+        take_dic = True
```

Comment:
just check that way of simplify

for current_dict in input_list:
        if all(current_dict.get(key) and current_dict[key].strip() for key in must_exist_values):
            new_list.append(current_dict)

Reply:
it is simplified but not readable